src/app/state/movies/movies.selectors.ts

```diff
@@ -8,3 +8,7 @@ export const selectMovie = createSelector(
   state => state.movies,
 );
 
+export const selectLoading = createSelector(
+  selectMovieState,
+  state => state.loading,
```

Comment:
jakbyś chciał jednak robić loading na każdym filmmie z osobna to niepotrzebny Ci będzie ten selecotr - ale co raz bardziej mi się wydaje że łatwiej Ci będzie jednak ojebać to loadingiem na state niż na filmie. Jesli tak sie zdecydujesz, to na Movie już nie powinno być flagi loading, a zostaw na state movies i w komponencie wtedy musisz zrobić że if movies.loading pokaż że loading, a jeśli nie loading to dopiero pokaż div ngFor=movies$